jwst/datamodels/schemas/guider_stream.schema.yaml

```diff
@@ -0,0 +1,61 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/guider_raw.schema"
+allOf:
+- $ref: core.schema
+- $ref: guider_meta.schema
+- $ref: bunit.schema
+- $ref: wcsinfo.schema
+- type: object
+  properties:
+    data:
+      title: The science data
+      fits_hdu: SCI
+      default: 0
+      ndim: 3
+      datatype: uint16
```

Comment:
One note: the ndim listed here is 3, and all stream files provided in the data directory have NAXIS = 3, but the TR that you provided for documentation would appear to indicate that the gs-acq streams would have NAXIS = 4. That might just be out of date. The ticket that spawned this issue (JWSTDMS-641) related to a gs-acq1_..._stream.fits file, for what it's worth.

Reply:
Good catch. The authors have been asked.